vql_plugins/server.go

```diff
@@ -29,6 +29,7 @@ import (
 	_ "www.velocidex.com/golang/velociraptor/vql/server/downloads"
 	_ "www.velocidex.com/golang/velociraptor/vql/server/favorites"
 	_ "www.velocidex.com/golang/velociraptor/vql/server/flows"
+	_ "www.velocidex.com/golang/velociraptor/vql/server/humio"
```

Comment:
I wonder if it should be placed under a different directory? Currently vql/server are all plugins related to manage the server. Maybe vql/tools/ ?

Technically this is not strictly required to be used on the server (although this probably where it is most useful).

Reply:
I don't have an opinion on where it is located. I chose server/ because that's where elastic and splunk are located. I can add a commit to move these as part of the updated PR.